src/main/kotlin/com/team4099/lib/pathfollow/TrajectoryUtils.kt

```diff
@@ -0,0 +1,89 @@
+package com.team4099.lib.pathfollow
+
+import com.pathplanner.lib.PathPlannerTrajectory
+import com.team4099.lib.geometry.Pose
+import com.team4099.lib.geometry.Translation
+import com.team4099.lib.units.LinearVelocity
+import com.team4099.lib.units.base.meters
+import com.team4099.lib.units.base.seconds
+import com.team4099.lib.units.derived.angle
+import com.team4099.lib.units.derived.radians
+import com.team4099.lib.units.inMetersPerSecond
+import com.team4099.lib.units.inMetersPerSecondPerSecond
+import com.team4099.lib.units.perSecond
+import edu.wpi.first.math.trajectory.TrajectoryParameterizer
+
+fun trajectoryFromPathfinder(pathPlannerTrajectory: PathPlannerTrajectory): Trajectory {
```

Comment:
oh also this is named wrong (should be `trajectoryFromPathPlanner` rather than Pathfinder)